app/src/components/common/network/index.tsx

```diff
@@ -68,7 +68,7 @@ export const Network: React.FC = props => {
       <ConnectionStatusWrapper>
         <ConnectionStatusDot />
         <ConnectionStatusText>
-          {truncateStringInTheMiddle(account) || 'No account connected'}
+          {truncateStringInTheMiddle(account, 41, 6, 4) || 'No account connected'}
```

Comment:
I don't remember the reason for the second parameter. I think just indicating the length of the first part and the second part should be enough: then if the length of the string is less than or equal to the sum of both arguments, you just return it.

For example:

```javascript
truncate("foobarbaz", 3, 2) // "foo...az"
truncate("foobar", 4, 2) // "foobar"
```

@mariano-aguero do you agree?

@hpmaxi can you do this (and also add tests for it)?

Reply:
Yes, I agree, we don't need the second parameter

Reply:
I hope there's enough tests